### ISSUE TITLE
InDesign Interactive Buttons not working

### ISSUE THREAD

**Issue by giannisftaras** [mozilla/pdf.js#12746]
Attach (recommended) or Link to PDF file here: https://drive.google.com/file/d/1ELjxPTwefDjiafHi_RwtNQH19RYc1dV5/view?usp=sharing

Configuration:
- Web browser and its version: Any
- Operating system and its version: Any
- PDF.js version: 2.6.347
- Is a browser extension: No

Steps to reproduce the problem:
1. Load any PDF that contains an interactive button that was created using Adobe InDesign.

What went wrong?
Is there a way to add support or implement the usage of interactive buttons inside PDF.js? If you load the example file on Adobe Acrobat Reader or the embedded Google Chrome PDF viewer the buttons appear correctly and can be pressed. PDF.js on the other hand renders the buttons as simple images on the canvas.
The test PDF was created using Adobe InDesign with the use of Interactive > Buttons and forms.

**Comment by timvandermeij** [comment]
In the most recent development version (at https://mozilla.github.io/pdf.js/web/viewer.html), I see `Warning: Push buttons without action dictionaries are not supported` in the console.

**Comment by giannisftaras** [comment]
You are correct. I accidentally pasted the console log from another similar PDF that I was having the same issue with.
https://drive.google.com/file/d/1bYatnJ_CcVv5vQ-oV-yueV6iARE9wfUN/view?usp=sharing

The issue though still remains the same. The buttons are rendered as images and can not be pressed.
